Harvest.Web/Controllers/Api/TicketController.cs

```diff
@@ -23,6 +23,26 @@ public TicketController(AppDbContext dbContext, IUserService userService)
             this._userService = userService;
         }
 
+        // create a new ticket via react
+        [HttpGet]
+        [Authorize(Policy = AccessCodes.PrincipalInvestigator)]
+        public async Task<ActionResult> List(int projectId, bool topOnly = true)

```

Comment:
I'd rather you optionally pass the number you want instead of `topOnly`.  it can be an optional field and if it's null get them all.  This was Cal can decide if he wants 3 or 5 or more and we only have to change one thing.